spec/findbugs_translate_checkstyle_format/translate_spec.rb

```diff
@@ -34,4 +102,44 @@
       end
     end
   end
+
+  describe 'set_dummy' do
+    context 'single SrcDir' do
+      xml = {
+        'BugCollection' => {
+          'Project' => {
+            'SrcDir' => 'test_dir'
+          }
+        }
+      }
+      require 'rexml/document'
+      doc = REXML::Document.new
+      checkstyle = doc.add_element("checkstyle")
```

Comment:
__[rubocop]__
* Prefer single-quoted strings when you don't need string interpolation or special symbols.